editor/script/script_text_editor.h

```diff
@@ -112,7 +112,7 @@ class ScriptTextEditor : public ScriptEditorBase {
 
 	PopupPanel *color_panel = nullptr;
 	ColorPicker *color_picker = nullptr;
-	Vector2 color_position;
+	Vector3 color_position;
```

Comment:
AFAICT it's used to store ints only.
```suggestion
	Vector3i color_position;
```